### ISSUE TITLE
Does not work with Jetifier

### ISSUE THREAD

**Issue by tadfisher** [mattprecious/swirl#9]
Jetifier breaks on transforming the 1.1.0 AAR:

```
Unable to resolve dependency for ':Mobile-Android@prodRelease/compileClasspath': Failed to transform file 'swirl-1.1.0.aar' to match attributes {artifactType=jetified-aar} using transform JetifyTransform
```

Looking at `swirl-1.1.0.aar` itself, it somehow contains two `AndroidManifest.xml` entries in the ZIP filesystem:

- AndroidManifest.xml

```
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.mattprecious.swirl"
    android:versionCode="1"
    android:versionName="1.0.1-SNAPSHOT" >

    <uses-sdk android:minSdkVersion="15" />

</manifest>
```

- AndroidManifest.xml

```
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.mattprecious.swirl"
    android:versionCode="1"
    android:versionName="1.1.0" >

    <uses-sdk android:minSdkVersion="14" />

</manifest>
```

**Comment by mattprecious** [comment]
Uuuuuuuhhhhhhh..... wtf?

**Comment by tadfisher** [comment]
I saw another project had the same issue: https://github.com/DreaminginCodeZH/MaterialRatingBar/issues/39

Looks like they just updated to the latest stable build tools and Android Gradle plugin and shipped again.

**Comment by tadfisher** [comment]
FYI you should not have the `<uses-sdk />` entry in the manifest as well, as that can conflict/override the consumer project's settings.

**Comment by JakeWharton** [comment]
uses-sdk won't cause conflicts unless your min is higher than the consumer's. it's special cased as it's the only way min and target (which you shouldn't specify) is propagated

**Comment by TylerMcCraw** [comment]
Looks like @jrodbx has got this fixed up in #10
I confirmed that PR fixes this issue for me.

**Comment by mattprecious** [comment]
1.1.1 has been released with the new build tools. Thanks for reporting and thanks @jrodbx for handing the updates!
